easybuild/easyconfigs/b/bcgTree/bcgTree-1.2.1-foss-2023a.eb

```diff
@@ -20,6 +20,10 @@ dependencies = [
     ('RAxML', '8.2.13', '-avx2'),
 ]
 
+# fix incompatibility with MUSCLE 5.x
+# see https://github.com/molbiodiv/bcgTree/issues/48
+prebuildopts = r"sed -i 's/\(muscle.*\) -in /\1 -align /g' lib/bcgTree.pm && "
```

Comment:
This is not actually run, because the `Tarball` easyblock is used...

It has to be moved to `postinstallcmds`...

Reply:
fixed in 608956e049